system/jlib/jstats.cpp

```diff
@@ -999,6 +999,11 @@ static const constexpr StatisticMeta statsMetaData[StMax] = {
     { PEAKNUMSTAT(PeakCacheObjects), "High water mark for number of objects in a cache"},
     { NUMSTAT(CacheDuplicates), "The number of times an item was added to a cache by two threads at the same time" },
     { NUMSTAT(CacheEvictions), "The number of times an item was evicted from a cache" },
+    { NUMSTAT(DuplicateKeyCount), "The number of duplicate keys" },
+    { SIZESTAT(OffsetBranches), "The 1st branch node offset position in the index" },
+    { SIZESTAT(BranchMemory), "The size of branch memory" },
```

Comment:
Should be "The estimated size of the branch nodes when stored in memory"

same for leaf nodes below.